nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateRecord.java

```diff
@@ -179,6 +194,27 @@
         .description("If the records cannot be read, validated, or written, for any reason, the original FlowFile will be routed to this relationship")
         .build();
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+
+    private static Validator createAttributeNameValidator() {
```

Comment:
I don't believe this validator is needed. Attribute names can be anything except for null and empty string. There does exist a validator for this already: StandardValidators.ATTRIBUTE_KEY_PROPERTY_NAME_VALIDATOR.